### ISSUE TITLE
'dlib/opencv.h' file not found

### ISSUE THREAD

**Issue by JanathonL** [alex011235/FaceSwap_iOS#1]
Although I have added libdlib.a  to frameworks, the xcode still doesn't build successfully and I get a error: 'dlib/opencv.h' file not found in the FaceSwap_iOS-master/FaceSwap/FSImageUtils.mm. It is the line I include <dlib/opencv>.
I have tried put the dlib library in the same folder with the FSImageUtils.mm and it works. But when I want to build it in my iPad, it generates the new error:
Apple Mach-O Linker (ld) Error Group
1."_USER_ERROR__missing_dlib_all_source_cpp_file__OR__inconsistent_use_of_DEBUG_or_ENABLE_ASSERTS_preprocessor_directives", referenced from:
2.clang: error: linker command failed with exit code 1 (use -v to see invocation)

Any help will be appreciated!

**Comment by alex011235** [comment]
Hmm, I've seen that somewhere else. Are you sure you have added OpenCv to it? I think it should be enough to add the unziped version of opencv2.framework.zip to frameworks.

**Comment by JanathonL** [comment]
Yes, I unzip the opencv framework and add it to the framework but the error is about the libdlib.a. It seems xcode does not correctly recognize the dlib because the error is can't find the file "dlib/opencv"

**Comment by alex011235** [comment]
When I added the dlib framework I just dropped it manually in the root. An I think I chose "copy ... if needed" (I don't remember clearly). 
I also think you might have to change some setting from debug to run for successfully running it on a device.
